src/README.md

```diff
@@ -70,9 +70,7 @@ Read more about variable formatting options in the [Variables](https://grafana.c
 
 ### Alerting
 
-See the [Alerting](https://grafana.com/docs/grafana/latest/alerting/alerts-overview/) documentation for more on Grafana alerts.
-
-[Unified Alerting](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/) queries should contain a time series field. Queries without this field will return an error: "input data must be a wide series but got type long". To return time series, you can use the [`CREATE_TIME_SERIES` function](https://docs.aws.amazon.com/timestream/latest/developerguide/timeseries-specific-constructs.views.html). For example:
+[Alerting](https://grafana.com/docs/grafana/latest/alerting/) queries should contain a time series field. Queries without this field will return an error: "input data must be a wide series but got type long". To return time series, you can use the [`CREATE_TIME_SERIES` function](https://docs.aws.amazon.com/timestream/latest/developerguide/timeseries-specific-constructs.views.html). For example:
```

Comment:
honestly not sure if this is still a constraint but I do know `unified alerting` is just alerting now 😬 [see this video for some fun historical context on history of alerting at Grafana](https://www.youtube.com/watch?v=7z5FwdfL7v4)